corehq/form_processor/submission_post.py

```diff
@@ -163,9 +162,6 @@ def run(self):
                 if instance.xmlns == DEVICE_LOG_XMLNS:
                     try:
                         process_device_log(self.domain, instance)
-                    except ProtocolError:
-                        # if riak is down/struggling, the caller will catch and notify
-                        raise
```

Comment:
Good question, I checked and as long as `notify_exception` is called within an `except` block then the `exec_info` should get passed through (also tested locally)